commands/mv.ts

```diff
@@ -32,24 +31,23 @@ exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
-  const { srcPath, destPath } = options;
-  const accountId = getAccountId(options);
+  const { srcPath, destPath, account } = options;
 
-  trackCommandUsage('mv', null, accountId);
+  trackCommandUsage('mv', null, account);
 
   try {
-    await moveFile(accountId, srcPath, getCorrectedDestPath(srcPath, destPath));
+    await moveFile(account, srcPath, getCorrectedDestPath(srcPath, destPath));
     logger.success(
       i18n(`${i18nKey}.move`, {
-        accountId,
+        account,
```

Comment:
These should all be `accountId: account` (or update the lang file to use new keys)

Reply:
Thanks for catching these! I was originally coding the `i18n` and `logger` functions incorrectly and then realized my mistake. I think these were some of the first files I was editing 🙏 

Addressed in https://github.com/HubSpot/hubspot-cli/pull/1212/commits/11c8c119479c484027f71dffa1ad97d64951d2d8